README.md

```diff
@@ -116,6 +146,92 @@ If the config is done correctly the test result should look like the screeshot b
 
 Click ***Save endpoint*** and you're done!.
 
+You can find more information on creating a github repository dispatch event via REST API [here](https://docs.github.com/en/rest/repos/repos#create-a-repository-dispatch-event)
+
+---
+
+## Configure Webhook Endpoint to trigger workflow in a branch
+To configure a webhook endpoint to trigger a workflow in a specific branch follow these steps
+
+* In the settings page scroll down to the section **WEB HOOKS** and click the Add endpoint button, this will show a modal where the endpoint values can be entered
+* Set the http method to *POST* and enter the endpoint url as follows `https://api.github.com/repos/{owner}/{repo}/actions/workflows/{workflow_id}`. The following values `{owner}`, `{repo}` and `{workflow_id}` should be replaced with the correct values. The `workflow_id` can be the id of the workflow you want to trigger or the name of the workflow instead.
+To get the list of workflows for a repository a GET request should be made to the following endpoint
+https://api.github.com/repos/OWNER/REPO/actions/workflows, note that `OWNER` and `REPO` should be replaced with the correct values. More information on this endpoint can be found [here](https://docs.github.com/en/rest/actions/workflows#list-repository-workflows). Find a sample curl request and response below
```

Comment:
Could you please put https://api.github.com/repos/OWNER/REPO/actions/workflows in ` ` so it's consitent with rest of the page?